### ISSUE TITLE
CONTAINER_POST_INIT_COMMAND 'for' loop seems to break due to word splitting

### ISSUE THREAD

**Issue by radarsymphony** [tiredofit/docker-alpine#58]
Hi tiredofit, 

I am running into errors trying to use your `CONTAINER_POST_INIT_COMMAND`. I think that the [**for loop**](https://github.com/tiredofit/docker-alpine/blob/16bf443f813778dba5c9d7d61d14404a758c7805/install/etc/cont-init.d/99-container#L130) you have to process and execute `CONTAINER_POST_INIT_COMMAND` might be breaking due to word splitting. Do I need to formatting the command in a particular way to use this env var in the docker-compose.yml? 

Initially, I had a more complex command, but even when I pared it down to `CONTAINER_POST_INIT_COMMAND=echo 'hello world'` the docker-compose logs show:

``` 
app        | /etc/cont-init.d/99-container: line 133: 'hello: command not found
app        | /etc/cont-init.d/99-container: line 133: world': command not found
```

From my reading of the [gnu bash manual](https://www.gnu.org/software/bash/manual/html_node/Looping-Constructs.html#Looping-Constructs), **for loops** in bash iterate over words by default, so I am interpreting the above log as bash trying to run each section of the command divided on whitespace.

Would a **while loop** work instead? I'm imagining something like the following:

```
# CONTAINER_POST_INIT_COMMAND="echo 'hello world'"

if [ -n "${CONTAINER_POST_INIT_COMMAND}" ] ; then
#  print_debug "Found Post Init Command"
  post_commands=$(echo "${CONTAINER_POST_INIT_COMMAND}" | tr "," "\n")
	while IFS= read -r post_command
	do
#	   print_debug "Executing post init command '${post_command}'"
	  eval "${post_command}"
	done <<< "${post_commands}"
fi
```
I am not sure if `eval` fits with your use case or if something like `set -- $post_command; "$@"` would be preferable instead.

**Comment by tiredofit** [comment]
Close - We can stick with the for loop, but there are a few things missing out of this command

- We need to quote the post_commands variable
- As you have identified, eval was also missing, which without it would simply list the command, and not execute it. 

This is fixed in upcoming `7.7.1`

**Comment by tiredofit** [comment]
https://github.com/tiredofit/docker-alpine/commit/728611bf06aa8ef084fa6fec54e4edca5fb900c0
